contrib/ci/dependencies.xml

```diff
@@ -764,14 +764,6 @@
     <distro id="fedora">
       <package>libgusb-devel</package>
     </distro>
-    <distro id="debian">
-      <control>
-        <version>(>= 0.3.3)</version>
-      </control>
-      <package variant="x86_64" />
-      <package variant="s390x">libgusb-dev:s390x</package>
-      <package variant="i386" />
-    </distro>
```

Comment:
after rebasing on #2340 this section can drop, but bump up the version to 0.3.5 so that it triggers CI to skip packaging with the new stuff.